src/Console/src/Sequence/CallableSequence.php

```diff
@@ -36,7 +33,6 @@ public function __construct(
         string $footer = ''
     ) {
         $this->function = $function;
-        $this->parameters = $parameters;
```

Comment:
only remove property and its assignment in constructor for unused private property without removing its middle `__construct` parameter that fill it to avoid BC break.